src/iotjs_module_buffer.cpp

```diff
@@ -82,6 +82,17 @@ size_t BufferWrap::length() {
 }
 
 
+static size_t BoundRange(int index, size_t low, size_t upper) {
```

Comment:
The naming is a bit out-of-style: are we not starting function and method names with lower case letters, reserving upper case first letters for classes? (this would be `boundRange` then)